scripts/mac_refresh.py

```
@@ -0,0 +1,92 @@
+        shutil.copyfile(cookie_path, temp_db)
+        try:
+            conn = sqlite3.connect(temp_db)
+            cursor = conn.cursor()
+            cursor.execute("SELECT encrypted_value, value FROM cookies WHERE name='d' AND host_key LIKE '%slack.com%'")
+            row = cursor.fetchone()
+            if row:
+                decrypted = decrypt_payload(key, row[0]) if row[0] else row[1]
+                if decrypted:
+                    d_cookie = urllib.parse.unquote(decrypted).encode('latin-1', 'ignore').decode('latin-1')
+            conn.close()
+        except: pass
```

Ruff (0.15.15)

[error] Do not use bare `except`

(E722)